kse/src/org/kse/gui/dialogs/extensions/resources_fr.properties

```diff
@@ -2,8 +2,8 @@
 
 
 
-DViewExtensions.ViewCert.Title=D�tails du certificat pour {0}
-DViewExtensions.ViewCrl.Title=D�tails de la liste de r�vocation pour {0}
+DViewExtensions.ViewCert.Title=D�tails du certificat pour {0}
```

Comment:
Encoding

Reply:
I've fixed it and tested. Should work nicely :)